tests/search.spec.ts

```diff
@@ -22,12 +22,25 @@ test("search", async ({ page }) => {
   const searchResultsHeading = page.getByRole("heading", { name: "Search results" });
   const searchResults = page.locator("div").filter({ has: searchResultsHeading }).last();
   await expect(searchResults).toBeVisible();
+
+  /** Check the structure of the search result */
+  const firstSearchResult = searchResults.getByRole("listitem").first();
+  /** heading */
+  await expect(firstSearchResult.getByRole("heading")).toBeVisible();
+  /** description */
+  await expect(firstSearchResult.getByRole("paragraph")).toBeVisible();
+
+  /** TODO: Make the markup more semantic */
```

Comment:
I've copied this from the [old cypress tests](https://github.com/climatepolicyradar/navigator-frontend/blob/bcb853b8f411ab493b0c69e1cb2cc78db9a2f32f/e2e/cypress/e2e/cclw/journeys/search.cy.js#L43-L51) as it felt like a valuable and I want to delete that test without losing any coverage.